README.rst

```diff
@@ -105,6 +103,17 @@ news and updates.
 .. _its Web site: https://beets.io/
 .. _@b33ts: https://twitter.com/b33ts/
 
+Contact
+-------
+* Encountered a bug you'd like to report or have an idea for a new feature? Check our our `issue tracker`_! If your issue or feature hasn't already been reported, please `open a new ticket`_ and we'll be in touch with you shortly.
```

Comment:
I'm not sure if you do this already internally, but it might be worth defining a way to sort feature requests. Something as simple as give a 👍 on features you'd most like to see implemented (at the cost of implementing other features). This way we can sort by 👍 on the issue tracker and go from there. Also hopefully cuts down on annoying +1s...

Reply:
Yes, great idea! Explicitly recommending that folks search the issue tracker for similar issues before opening a new one (and voting!) seems very helpful.